lib/hawk.rb

```diff
@@ -1,8 +1,11 @@
 ##
 # Hawk entry point.
 #
+#
+
+require 'active_support/core_ext/array/wrap'
```

Comment:
Is this enough? Or, why is it needed? We are using `pluralize` but not requiring it explicitly.